components/NewsCard.js

```diff
@@ -44,10 +44,10 @@ export default function NewsCard({
                 width={50}
                 height={50}
               ></Image>
-              <StyledProviderName>{providerName}</StyledProviderName>
+              <StyledP>{providerName}</StyledP>
             </StyledDiv>
             <p>{moment(datePublished).startOf("ss").fromNow()}</p>
-          </StyledProvider>
+          </StyledContainer>
         </StyledA>
       </Card>
     </>
```

Comment:
You don't need a fraction `<>` element at this point. Your `<Card>` is already your top level element.

Reply:
done!